server/config/production.json

```diff
@@ -5,5 +5,8 @@
   "csp": {
     "enabled": true,
     "reportOnly": true
+  },
+  "babel": {
```

Comment:
do you think we can find a way to fix this without making in all of our configs?